.circleci/config.yml

```diff
@@ -24,7 +24,7 @@ base_job: &base_job
     XCODE_VERSION: *xcode_version
 
 jobs:
-  swiftlint-and-cache:
+  build-and-cache:
```

Comment:
This renaming will require tweaking our `master` branch protection rule but since this step is no longer going to be using `Swiftlint` we should use a better naming